aten/src/ATen/NamedTensor.h

```diff
@@ -71,9 +71,9 @@ struct CAFFE2_API NoNamesGuard {
 CAFFE2_API Tensor& internal_set_names_inplace(Tensor& tensor, optional<DimnameList> names);
 CAFFE2_API Tensor& internal_set_names_inplace(Tensor& tensor, std::vector<Dimname>&& names, bool validate_names);
 
-// Everywhere this is used, it is possible to not instantiate the vector by doing
-// some more clever bookkeeping. This is important for performance.
-std::vector<Dimname> FIXME_default_names(size_t len);
+constexpr size_t kMaxNamedTensorDim = 64;
```

Comment:
it seems like there should be a more generic constant, since 64 max dims is true regardless of named tensors. Maybe we could add a global constant and reference it both in named tensors and in places like WRapDimUtilsMulti.h ?

Reply:
Sure. I'd need to go audit those places and figure out if there's a way to explain the limitation. I think that that task is outside the scope of this PR; I have an issue open for it here: https://github.com/pytorch/pytorch/issues/23159

Reply:
@nairbv Aside from having a more generic constant, do you have other comments on this PR? I'd prefer to punt figuring out the (undocumented) dimensionality constraints of a Tensor to later because it isn't very related to this stack of PRs.

Reply:
that was my only concern, yup it's fine to refactor later